css/Sass/profile.scss

```diff
@@ -0,0 +1,22 @@
+
+.profile {
+	h1 {
+		font-size: 48px;
+		position: initial;
+			small {
```

Comment:
It looks like a lot of the same style is already defined in main.scss. Is this necessary?